extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/directives/directive.configDescription.js

```diff
@@ -3,6 +3,9 @@ angular.module('PaperUI.directive.configDescription', []) //
 
     var controller = function($scope) {
         $scope.getName = function(parameter, option) {
+            if (!option) {
+                return undefined;
```

Comment:
not sure if your ESLint configuration allows it but simple `return;` should suffice as it returns `undefined` under the hood